packages/office-ui-fabric-react/src/utilities/positioning.ts

```diff
@@ -1,17 +1,16 @@
 import { DirectionalHint } from '../common/DirectionalHint';
 import {
-  IRectangle,
-  assign,
   getScrollbarWidth,
-  Rectangle,
   getRTL,
+  Rectangle as FullRectangle,
+  IRectangle
 } from '../Utilities';
 
 export enum RectangleEdge {
```

Comment:
these can be const enum to reduce bundle size

Reply:
No they can't I use the string as well, it needs to stay as a regular enum.